services/cd-service/pkg/repository/transformer_test.go

```diff
@@ -790,9 +790,58 @@ func TestReleaseTrainErrors(t *testing.T) {
 func TestRbacTransformerTest(t *testing.T) {
 	tcs := []struct {
 		Name          string
+		ctx           context.Context
 		Transformers  []Transformer
 		ExpectedError string
 	}{
+		{
+			Name: "able to create undeploy with permissions policy",
+			Transformers: []Transformer{
+				&CreateEnvironment{
+					Environment: "staging",
+				},
+				&CreateEnvironment{
+					Environment: "production",
+					Config:      config.EnvironmentConfig{Upstream: &config.EnvironmentConfigUpstream{Environment: "staging", Latest: true}},
+				},
+				&CreateApplicationVersion{
+					Application: "app1",
```

Comment:
```suggestion
					Application: "app1",
					Authentication: nil,
```

Reply:
`Authentication: nil` is not possible. I added `Authentication{RBACConfig: auth.RBACConfig{DexEnabled: false}}`.